go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/mdlayher/raw v0.0.0-20191009151244-50f2db8cc065 // indirect
 	github.com/rck/unit v0.0.3
 	github.com/rekby/gpt v0.0.0-20200219180433-a930afbc6edc
+	github.com/rminnich/go-lzo v0.0.0-20201110235843-e5cfc6a1d9af
```

Comment:
Why is this lzo stuff added?

Reply:
no idea. It used to be used. Maybe we need to run go mod tidy on this again?